### DISCUSSION TITLE
issues connecting nextPYP to remote cluster

### DISCUSSION THREAD

**Discussion by chelseanfries** [nextpyp/.github#115] [Q&A] [answered]
Hi, I have installed nextPYP locally for my computer that is on a shared network, but am having problems connecting to our compute cluster. Local jobs run just fine, but when I try to run a job which relies on remote cpu, they always fail. From my cluster, I can see a job named 'pyp_launch' which fails, and my error code in the log file reads `ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /pyp (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f66bdc29eb0>: Failed to establish a new connection: [Errno 111] Connection refused'))` Has anyone had this issue before? thanks!

**Comment by cuchaz** [comment, marked as answer]
Hi there.

From that error message, it seems that the cluster job isn't able to reach the web server. Without knowing more about your specific networking environment, it's hard to suggest which changes to the configuration you'd need to make to fix this, but the [Enable remote access](https://nextpyp.app/files/pyp/latest/docs/install/enable-remote-access.html) or the [Attach a SLURM cluster](https://nextpyp.app/files/pyp/latest/docs/install/attach-slurm.html) parts of the installation instructions may be able to help.

In particular, I'd pay attention to the `web.host` and `web.webhost` settings since they're the most relevant in solving these kinds of issues.

If you want to share details of how your network is set up and your configuration file, then I can try to give more specific suggestions.
